dnf-plugins-core.spec

```diff
@@ -150,14 +153,17 @@ Obsoletes:      python3-dnf-plugins-extras-leaves < %{dnf_plugins_extra}
 %description -n python3-dnf-plugin-leaves
 Leaves Plugin for DNF, Python 3 version. List all installed packages
 not required by any other installed package.
+%endif
 
 %package -n python2-dnf-plugin-local
 Summary:        Local Plugin for DNF
 Requires:       %{_bindir}/createrepo_c
 Requires:       python2-%{name} = %{version}-%{release}
+%if !%{with python3}
 Provides:       dnf-plugin-local =  %{version}-%{release}
-Provides:       python2-dnf-plugins-extras-local = %{version}-%{release}
 Provides:       dnf-plugins-extras-local = %{version}-%{release}
+%endif
+Provides:       python2-dnf-plugins-extras-local = %{version}-%{release}
```

Comment:
what about providing `python-dnf-plugins-extras-local` as well? it has been done in extras by `%python_provide`